src/pages/ProblemEditor/MarkdownEditor/MarkdownEditor.tsx

```diff
@@ -1,17 +1,109 @@
 import { Divider, Stack, Typography } from "@mui/material";
 import React from "react";
-import ReactMarkdown from "react-markdown";
 import Editor from "@monaco-editor/react";
 import { FormikProps } from "formik";
+import { isBlank } from "../../../shared/utils";
+import { Markdown } from "../../../shared/Markdown";
 
 interface Props {
   form: FormikProps<any>;
 }
 
+const defaultMarkdown = `[comment]: # (The title is auto generated to be at the top of problem descriptions.)
+
+## Problem Statement
+Type your problem description here
```

Comment:
Yeah, I think doing file I/O is probably harder than necessary. But exporting it from its own file might be fine. Even if its only used here. The idea is to separate it from all the `.jsx` code files. 

That way you can have the markdown separate from the logic of these files, in case the file is changed later. One can just import that default text afterwards. It makes the code more modular.

Now that I think about it, it would have been nice to do that too to the header/body/footer default text as well, but that is a different issue.